osclib/stagingapi.py

```diff
@@ -998,16 +998,17 @@ def map_ring_package_to_subject(self, project, pkg):
         """
         # it's actually a pretty stupid algorithm, but it might become more complex later
 
-        if project.endswith(':DVD'):
-            return project  # not yet
+        # assuming it is in adi staging, workaround for https://progress.opensuse.org/issues/9646
+        if self.is_adi_project(project) and self.item_exists(project, pkg):
```

Comment:
Does the `self.item_exists(project, pkg)` check make sense? This code is called during select (`submit_to_prj()`) which means the package likely does not exist in the staging yet. Not to mention the whole point is to avoid `:DVD` when using an `adi` regardless of the state of the `adi`.

I realize it was there before, but not exactly sure how it works properly.